Assets/MRTK/Core/Inspectors/Profiles/MixedRealityPointerProfileInspector.cs

```diff
@@ -62,41 +67,46 @@ public override void OnInspectorGUI()
             {
                 serializedObject.Update();
 
+                EditorGUILayout.Space();
+                EditorGUILayout.PropertyField(pointingExtent);
+                EditorGUILayout.PropertyField(pointingRaycastLayerMasks, RaycastLayerMaskContent, true);
+                EditorGUILayout.PropertyField(pointerMediator);
+                EditorGUILayout.PropertyField(primaryPointerSelector);
+
+                GUIStyle boldFoldout = new GUIStyle(EditorStyles.foldout) { fontStyle = FontStyle.Bold };
+
                 EditorGUILayout.Space();
                 EditorGUILayout.LabelField("Gaze Settings", EditorStyles.boldLabel);
                 {
                     EditorGUILayout.Space();
-                    EditorGUILayout.PropertyField(gazeCursorPrefab, new GUIContent("Gaze Cursor Prefab"));
+                    EditorGUILayout.PropertyField(gazeCursorPrefab, GazeCursorPrefabContent);
                     EditorGUILayout.PropertyField(gazeProviderType);
                     EditorGUILayout.PropertyField(useHeadGazeOverride);
                     EditorGUILayout.PropertyField(isEyeTrackingEnabled);
                     EditorGUILayout.Space();
 
-                    if (InspectorUIUtility.RenderIndentedButton("Customize Gaze Provider Settings"))
+                    var gazeProvider = CameraCache.Main.GetComponent<IMixedRealityGazeProvider>();
+                    CreateCachedEditor((Object)gazeProvider, null, ref gazeProviderEditor);
+
+                    showGazeProviderProperties = EditorGUILayout.Foldout(showGazeProviderProperties, "Gaze Provider Settings", true, boldFoldout);
+                    if (showGazeProviderProperties)
```

Comment:
We should also null check gazeProviderEditor somewhere in here. if `gazeProvider` is null, the resulting editor will be too

Reply:
Done